core/src/main/scala/com/cloudera/livy/sessions/SessionState.scala

```diff
@@ -61,19 +61,19 @@ object SessionState {
     override def toString = "shutting_down"
   }
 
-  case class Error(time: Long = System.currentTimeMillis()) extends SessionState {
+  case class Error(time: Long = System.nanoTime()) extends SessionState {
```

Comment:
Is `SessionState` exposed to clients? If it is, using `nanoTime` here is kinda useless, since `nanoTime` only makes sense on the machine where it was called.